node_exporter/README.md

```diff
@@ -0,0 +1,3 @@
+Note that while this installs `node_exporter`, you will also need to
+make sure our firewall opens up port 9100 to our Prometheus server
+before it will be able to poll it.
```

Comment:
Is it worth adding a UFW task in this role to open port 9100 to Prometheus' IP address? Or does that make things more complex in terms of making sure UFW is installed before this role is run?

I think our UFW usage is pretty standard across Open edX and AVL.

Reply:
Our Open edX deployments (and upstream as well) actually don't use UFW at all (AMC being the exception). All the firewall config is at the GCP level.

Reply:
I'm rethinking my previous comment. We'd likely need to open the port in GCP's firewall as well, which would happen in Terraform. Let's leave the responsibility for the UFW and Terraform changes to the person using this role.

Reply:
Yeah, firewall stuff is always kind of hard to coordinate. I'd rather have the default be that when this role gets included, it's still blocked by the firewall and specific action has to be taken to set that up, whitelisting the prometheus IP address. Longer term, my hope is to have a dedicated Prometheus/Grafana instance in each deployment so it would be accessing over a local IP and wouldn't require additional firewall config.